UnitThree/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <tab id="rltsquare2" translate="label" sortOrder="50">
+            <label>RLTsquare</label>
+        </tab>
+        <section id="rltsquare2" translate="label" type="text" sortOrder="40" showInDefault="1" showInWebsite="1"
+                 showInStore="1">
+            <class>separator-top</class>
+            <label>Unit Three</label>
+            <tab>rltsquare2</tab>
+            <resource>RLTSquare_UnitThree::UnitThree_config</resource>
+            <group id="general" translate="label" type="text" sortOrder="20" showInDefault="1" showInWebsite="1"
+                   showInStore="1">
+                <label>Settings</label>
+                <field id="enable" translate="label" type="select" sortOrder="1" showInDefault="1" showInWebsite="0" showInStore="1">
+                    <label>Module Enable</label>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                </field>
+                <field id="username" translate="label comment" type="text" sortOrder="30"
+                       showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>User Name :</label>
+                </field>
+                <field id="password" translate="label comment" type="obscure" sortOrder="40"
```

Comment:
Password and username fields should have been set for each environment ... like staging_password, production_password ... then use the <depends> to hide/show it based of environment type. We want it so that when we switch environment, we have the user/pass stored specific to each env,